composer.json

```
@@ -19,6 +19,7 @@
         "ext-zip": "*",
         "erusev/parsedown-extra": "^0.8.1",
         "filp/whoops": "^2.7",
+        "maennchen/zipstream-php": "^2.4",
```

Now that PHP versions <= 8.1 are [end of life](https://www.php.net/supported-versions.php) we can bump this to v3.

```suggestion
        "maennchen/zipstream-php": "^3.1",
```

---

Alright. It will need refactoring then because they integrated the size calculation beforehand inside the library. So I will remove quite a lot of code and simplify the whole process :)